modules/abstract-utxo/src/abstractUtxoCoin.ts

```diff
@@ -1534,7 +1534,12 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
   }
 
   async presignTransaction(params: PresignTransactionOptions): Promise<any> {
-    if (params.bulk && params.allowNonSegwitSigningWithoutPrevTx === undefined) {
```

Comment:
It does not make sense to keep the check for `params.bulk` because it doesn't really make sense in retrospect